clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -183,6 +229,15 @@ public ListTopicsResult listTopics() {
      * This is a convenience method for #{@link AdminClient#describeTopics(Collection, DescribeTopicsOptions)} with
      * default options. See the overload for more details.
      *
+     * <p>The following exceptions can be anticipated when calling {@code get()} on the futures obtained from the
+     * {@link DescribeTopicsResult#values() values()} method of the  returned {@code DescribeTopicsResult}</p>
+     * <ul>
+     *     <li>{@link org.apache.kafka.common.errors.InvalidTopicException}
+     *     if topic name collides with other topic name</li>
```

Comment:
InvalidTopicException happens when the topic name can't be represented in the request, not if it collides with another topic name.